svcs/amigo/table.go

```diff
@@ -33,40 +36,47 @@ type TeamRow struct {
 }
 
 type Scoreboard struct {
-	Chals 	[]ChalRow	`json:"challenges"`
+	Chals 	[]ChalCategory	`json:"challenges"`
 	TeamRow []TeamRow	`json:"teams"`
 }
 
 func (fd *FrontendData) initTeams(teamId string) []byte {
 
 	teams := fd.ts.GetTeams()
 	rows := make([]TeamRow, len(teams))
-	challenges := make([]ChalRow, 5)
-
+	var challenges []ChalCategory
+	var realChallenges []ChalCategory
```

Comment:
why we have same things with different names ?

Reply:
That's quite important. basically i remove the categories that have not any challenges inside.

Reply:
creating a new variable was the only way to do that

Reply:
but do we have categories which has no challenges ?